### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by daandemeyer**
This is all the stuff from #34628 that works

### CODE REVIEW

**Comment by DaanDeMeyer** [comment]
Setting the green label immediately since this was all reviewed and stamped in #34628 already

**Comment by bluca** [comment]
> This is all the stuff from #34628 that works

Computer says no: `82/92 systemd:integration-tests / TEST-70-TPM2       FAIL       91.63s   exit status 1`

**Review by yuwata — APPROVED** [review]

**Comment by yuwata** [comment]
Please rebase.

**Comment by yuwata** [comment]
Hm? You rebased an old branch??

**Comment by DaanDeMeyer** [comment]
> Hm? You rebased an old branch??

I ddi? I don't see what's wrong?

**Comment by yuwata** [comment]
> > Hm? You rebased an old branch??
> 
> I ddi? I don't see what's wrong?

Now the CodeQL linter complaining again.

**Comment by DaanDeMeyer** [comment]
> > > Hm? You rebased an old branch??
> > 
> > 
> > I ddi? I don't see what's wrong?
> 
> Now the CodeQL linter complaining again.

I didn't actually fix these as they're false positives. I remember @poettering didn't fix these either when he first introduced this logic.

File: src/ukify/ukify.py [hunk 1]
```diff
@@ -1801,7 +1801,14 @@ def create_parser() -> argparse.ArgumentParser:
...
 
-def finalize_options(opts: argparse.Namespace) -> None:
+def resolve_at_path(value: Optional[str]) -> str:
+    if value and value.startswith('@'):
+        return Path(value[1:])
+
+    return value
+
+
+def finalize_options(opts) -> None:
```
Comment (yuwata) [inline src/ukify/ukify.py]: Why the type annotation for `opts` is dropped?

**Pull request merged**